File: THEORY/MultistageBuildJava/hello.java
import java.util.Scanner;

public class hello {
    public static void main(String[] args) {
        String storedSapId = "500119435";
        Scanner scanner = new Scanner(System.in);

        while (true) {
            System.out.print("Enter your SAP ID: ");
            String userInput = scanner.nextLine();

            if (userInput.equals(storedSapId)) {
                System.out.println("Matched");
                break;
            } else {
                System.out.println("Not Matched");
            }
        }
        scanner.close();
    }
}

File: THEORY/Assignment/hello.c
#include <stdio.h>
#include <string.h>

int main() {
    char stored_sapid[] = "500119435";
    char user_input[20];

    while (1) {
        printf("Enter your SAP ID: ");
        scanf("%s", user_input);

        if (strcmp(user_input, stored_sapid) == 0) {
            printf("Matched\n");
            break; 
        } else {
            printf("Not Matched\n");
        }
    }
    return 0;
}